README.md

```diff
@@ -1707,6 +1723,8 @@ A [search engine](https://en.wikipedia.org/wiki/Search_engine_(computing)) is an
 
 **[`^        back to top        ^`](#)**
 
+All in one self hosting software.
```

Comment:
Needs a better description in my opinion (what does it do? `Software for easy installation, management and configuration of self-hosted services and applications`?)